### DISCUSSION TITLE
How can i enable dark mode in only one website?

### DISCUSSION THREAD

**Discussion by Dolaaz** [darkreader/darkreader#11158] [Q&A] [answered]
i just want to enable dark mode in a specific website, how can i?

**Comment by ngoclong19** [comment, marked as answer]
From Dark Reader setting, please go to Site list -> Inverted list only. See here:
`https://darkreader.org/help/en/`

**Reply by Dolaaz** [reply]
> From Dark Reader setting, please go to Site list -> Inverted list only. See here: `https://darkreader.org/help/en/`

thank you
